src/Nancy/Bootstrapper/NancyBootstrapperBase.cs

```diff
@@ -245,6 +245,8 @@ public void Initialise()
 
             this.ConfigureApplicationContainer(this.ApplicationContainer);
 
+            AppDomainAssemblyTypeScanner.UpdateTypes();

```

Comment:
Its fine. It fixed a bug in master we found last night. Decided its fine to include cos its only run on start up.

/approved.